Documentation/6.0.2/README.rst

```diff
@@ -0,0 +1,1518 @@
+    * dkms
+    * nuage-metadata-agent (from el7-6wind)
+    * nuage-openvswitch (from el7-6wind)
+    * python-pyelftools
+    * selinux-policy-nuage-avrs
+    * virtual-accelerator-base
+
+
+Deployment Process
+-------------------
+
+Phase 1: Install OpenStack director on the Undercloud system.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Follow the steps in https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/13/html/director_installation_and_usage/installing-the-undercloud .
+
+When obtaining images for the Overcloud nodes, replace the upstream Overcloud image with one modified to include Nuage components from Step 2 in this workflow.
+
+If you want to use a remote registry for the Overcloud container images, you need to add the iptables rule on Director (Undercloud Machine) with the Undercloud IP address or interface with external connectivity for NAT. In the below example, the Undercloud IP address is 192.168.24.1 and the external interface name is eth0:
+
+::
+
+    sudo iptables -A POSTROUTING -t nat -s 192.168.24.1/24 -j MASQUERADE
+    (or)
+    sudo iptables -t nat -A POSTROUTING -o eth0 -j MASQUERADE
+
+
+Phase 2: Configure the basic Overcloud.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Follow the upstream OpenStack documentation *up to the step where* the ``openstack overcloud deploy`` command is run using the CLI or starting the Overcloud deployment (starting the Overcloud creation) in the UI.
+
+These are the OpenStack instructions:
+
+    * Configuring the container registry details: https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/13/html/director_installation_and_usage/configuring-a-container-image-source
+    * Using the CLI: https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/13/html/director_installation_and_usage/chap-configuring_basic_overcloud_requirements_with_the_cli_tools
+    * Using the UI: https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/13/html/director_installation_and_usage/chap-configuring_basic_overcloud_requirements_with_the_ui_tools
+
+
+
+Phase 3: Modify the Overcloud qcow image (for example, overcloud-full.qcow2) to include Nuage components.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The steps for modifying overcloud-full.qcow2 are provided in the README.md file: https://github.com/nuagenetworks/nuage-ospdirector/tree/OSPD13/image-patching/stopgap-script/README.md .
+
+
+Phase 4: Adding Nuage Heat Templates
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Copy the nuage-tripleo-heat-templates folder form `nuage-tripleo-heat-templates <../../nuage-tripleo-heat-templates>`_ to `/home/stack/` directory on undercloud.
+
+.. Note:: As an end-user, it is a best practice to manage the templates using GIT. If you are aware of git workflow, you can run following command to manage your nuage-template change via GIT.
```

Comment:
Revise the text as follows.
From:
If you are aware of git workflow, you can run following command to manage your nuage-template change via GIT.
To:
You can run following command to manage the Nuage template change using GIT.

Reply:
Keep the text as is because this is a critical step.

Reply:
done